### PR TITLE
Add a .nvmrc file to support using the intended Node.js version

### PR DESCRIPTION
**Pull request by magdapoppins**
(Dev experience suggestion - feel free to close the pr if this doesn't seem like a good addition.)

As per https://cml.dev/doc/contributing/core, 16.x is the correct Node version to use for this repo - adding an .nvmrc file allows contributors that have nvm at hand to simply call `nvm use` to autoselect the projects intended Node version before `npm install`.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+16
```
Comment (dacbd) [inline .nvmrc:1]:
```suggestion
16

```

**Review by dacbd — APPROVED** [review]
added newline to end of file for my OCD

**Review by casperdcl — COMMENTED** [review]
but, but, but... https://github.com/iterative/cml/blob/65a200aa0042d8ce219933346741f9fb2348995f/package.json#L33

(my OCD hates dotfile root clutter)

**Comment by dacbd** [comment]
ㅋㅋㅋ we can revert if your OCD becomes too much ❤️

If it encourages another open source contributor my vote is +1.

**Pull request merged**